mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.purgeOrder = async (req, res) => {
+  try {
+    const result = await Order.findOneAndRemove({ _id: req.params.orderId });
+    if (!result) {
+      return res.status(404).json({ message: "Order not located" });
+    }
+    res.status(200).json({ message: "Order eradicated successfully" });
```

Comment:
The response message "Order eradicated successfully" is not following the standard HTTP response message convention. It should be more descriptive and follow a consistent tone throughout the application. Consider changing it to something like "Order deleted successfully" or "Order removal successful".